scripts/once-per-template.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-restricted-syntax, no-await-in-loop */
+import { Command } from 'commander';
+import execa from 'execa';
+
+import { getOptions, getCommand, getOptionsOrPrompt, createOptions } from './utils/options';
+import type { OptionSpecifier } from './utils/options';
+import { filterDataForCurrentCircleCINode } from './utils/concurrency';
+
+import TEMPLATES from '../code/lib/cli/src/repro-templates';
+
+export type Cadence = 'ci' | 'daily' | 'weekly';
+export type Template = {
+  name: string;
+  script: string;
+  cadence?: readonly Cadence[];
+  skipScripts?: string[];
+  // there are other fields but we don't use them here
+};
+export type TemplateKey = string;
+export type Templates = Record<TemplateKey, Template>;
+
+export async function parseCommand(commandline: string) {
+  const argv = commandline.split(' ');
+
+  const [yarn, scriptName] = argv;
+  if (yarn !== 'yarn') throw new Error('only works with scripts at this point');
```

Comment:
This could probably be a little clearer
```suggestion
  if (yarn !== 'yarn') throw new Error('This command only works with yarn scripts at this point');
```

Reply:
I'm getting this restriction in a later PR so I'll leave it.